routers/repo/repo.go

```diff
@@ -95,6 +95,14 @@ func checkContextUser(ctx *context.Context, uid int64) *models.User {
 		return nil
 	}
 	if !ctx.User.IsAdmin {
+		orgsAvailable := []*models.User{}
```

Comment:
Why was it added here? I don't see it being used anywhere else

Reply:
This appears to have been duplicated above (Line 65) during a merge.  Fixing that now, but other than that I'm not sure what you mean.